RELEASING.md

```diff
@@ -31,11 +31,7 @@ Label PRs with the next major version label (e.g. `2.0.0`) and merge changes int
 
 ### Plugin Branching
 
-Plugins, such as [job-scheduler](https://github.com/opensearch-project/job-scheduler) aren't as active as OpenSearch, and typically track 2 releases in parallel instead of 3. This still translates into 3 branches. For example, given the last major release of 1.0, job-scheduler maintains the following.
-
-* **main**: The _next_ release, currently 1.1. This is the branch where all merges take place, and code moves fast.
-* **1.x**: A common parent branch for the series of 1.x releases. This is where 1.x patches will be made when `main` becomes 2.0.
-* **1.0**: The _current_ release, currently 1.0. This branch's parent is `1.x` to make future merges easier. 'In between minor releases, only hotfixes (e.g. security) are backported to `1.0`. The next release out of this branch will be 1.0.1.
+Plugins are bundled and shiped together along with OpenSearch for every release. Plugin Branching should be same as OpenSearch which should allow working on 3 releases at the same time.
```

Comment:
Plugin branching follows OpenSearch core branching that will allow…

Reply:
Updated @bbarani